From b825c282a0d8570503e4fbf9dbda702230394401 Mon Sep 17 00:00:00 2001
From: Wenzong Fan <wenzong.fan@windriver.com>
Date: Sun, 30 Mar 2014 22:25:59 -0400
Subject: [PATCH] semanage: process ValueError for sepolicy, seobject

The sepolicy, seobject modules raise many unprocessed ValueError, just
process them in semanage to make the script proivdes error message but
not error trace.

Uptream-Status: Pending

Signed-off-by: Wenzong Fan <wenzong.fan@windriver.com>
Signed-off-by: Yi Zhao <yi.zhao@windriver.com>
---
 semanage/semanage | 7 ++++++-
 1 file changed, 6 insertions(+), 1 deletion(-)

diff --git a/semanage/semanage b/semanage/semanage
index 144cc00..7d93777 100644
--- a/semanage/semanage
+++ b/semanage/semanage
@@ -25,8 +25,13 @@
 
 import traceback
 import argparse
-import seobject
 import sys
+try:
+    import seobject
+except ValueError, e:
+    print "Error: %s\n" % e
+    sys.exit(1)
+
 PROGNAME = "policycoreutils"
 try:
     import gettext
-- 
2.7.4

